opencart 3.0.0.0 - 3.0.3.1/upload/admin/controller/extension/module/smaily_for_opencart.php

```diff
@@ -85,7 +85,33 @@ public function index() {
                 'name' => $this->language->get('section_abandoned'),
             ),
         );
+        // Get URL for CRON links.
+        $url = new Url(HTTP_CATALOG, $this->config->get('config_secure') ? HTTP_CATALOG : HTTPS_CATALOG);
```

Comment:
I think you have the shorthand if-condition the wrong way. At the moment it reads: if `config_secure` has been enabled return unsecure store URL, else return secure store URL.